src/summon/browzine-summon-adapter.js

```diff
@@ -398,54 +586,112 @@ browzine.summon = (function() {
+        }
+
+        if (browzineWebLink && browzineEnabled && isJournal(scope) && showJournalBrowZineWebLinkText()) {
+          var template = browzineWebLinkTemplate(scope, browzineWebLink);
+          $(documentSummary).find(".docFooter .row:eq(0)").append(template);
+        }
 
-      if(browzineWebLink && browzineEnabled && isArticle(scope) && showArticleBrowZineWebLinkText()) {
-        var template = browzineWebLinkTemplate(scope, browzineWebLink);
-        $(documentSummary).find(".docFooter .row:eq(0)").append(template);
+        if (browzineWebLink && browzineEnabled && isArticle(scope) && showArticleBrowZineWebLinkText()) {
+          var template = browzineWebLinkTemplate(scope, browzineWebLink);
+          $(documentSummary).find(".docFooter .row:eq(0)").append(template);
+        }
+
+        if (coverImageUrl && !defaultCoverImage && showJournalCoverImages()) {
+          $(documentSummary).find(".coverImage img").attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", "1px 1px 2px #ccc");
+        }
       }
 
-      if(coverImageUrl && !defaultCoverImage && showJournalCoverImages()) {
-        $(documentSummary).find(".coverImage img").attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", "1px 1px 2px #ccc");
+      if (request.readyState == XMLHttpRequest.DONE && request.status == 404) {
+        var endpoint = getUnpaywallEndpoint(scope);
+
+        if (endpoint && isUnpaywallEnabled()) {
+          var requestUnpaywall = new XMLHttpRequest();
+          requestUnpaywall.open("GET", endpoint, true);
+          requestUnpaywall.setRequestHeader("Content-type", "application/json");
+
+          requestUnpaywall.onload = function() {
+            if (requestUnpaywall.readyState == XMLHttpRequest.DONE && requestUnpaywall.status == 200) {
+              var response = JSON.parse(requestUnpaywall.response);
+
+              var unpaywallArticlePDFUrl = getUnpaywallArticlePDFUrl(response);
+              var unpaywallArticleLinkUrl = getUnpaywallArticleLinkUrl(response);
+              var unpaywallManuscriptArticlePDFUrl = getUnpaywallManuscriptArticlePDFUrl(response);
+              var unpaywallManuscriptArticleLinkUrl = getUnpaywallManuscriptArticleLinkUrl(response);
+
+              var template;
+
+              if (unpaywallArticlePDFUrl) {
+                template = unpaywallArticlePDFTemplate(unpaywallArticlePDFUrl);
+              }
+
+              if (unpaywallArticleLinkUrl) {
+                template = unpaywallArticleLinkTemplate(unpaywallArticleLinkUrl);
+              }
+
+              if (unpaywallManuscriptArticlePDFUrl) {
+                template = unpaywallManuscriptPDFTemplate(unpaywallManuscriptArticlePDFUrl);
+              }
+
+              if (unpaywallManuscriptArticleLinkUrl) {
+                template = unpaywallManuscriptLinkTemplate(unpaywallManuscriptArticleLinkUrl);
+              }
```

Comment:
Hey @tiandavis ,
Similar to the logic change I mentioned in Nomad in https://github.com/thirdiron/nomad/pull/110 , I think let's do something like `if... else if... else if...` here to make sure the code more clearly expresses that `unpaywallArticlePDFUrl` gets the top priority.

Thanks!